packages/server/graphql/intranetGraphQLHandler.ts

```diff
@@ -9,7 +9,7 @@ import uWSAsyncHandler from './uWSAsyncHandler'
 
 interface IntranetPayload {
   query: string
-  variables: object
+  variables: any
```

Comment:
curious why the change? any includes a primitive, which could break it

Reply:
Eslint gave the following error when using `object` type:
```
parabol-server: 12:14  error  Don't use `object` as a type. The `object` type is currently hard to use ([see this issue]((https://github.com/microsoft/TypeScript/issues/21732))).
parabol-server: Consider using `Record<string, unknown>` instead, as it allows you to more easily inspect and use the keys  @typescript-eslint/ban-types
```
I haven't been able to log the variables to see exactly what they are so I've updated `any` to be the suggested `Record<string, unknown>`, making sure that variables can't be primitives while still giving us flexibility in the object with `unknown`